src/napari_imagej/utilities/_module_utils.py

```diff
@@ -422,7 +402,7 @@ def _add_napari_metadata(
     # Without this, magicgui doesn't pick up on the types.
     type_hints = {str(i.getName()): python_type_of(i) for i in unresolved_inputs}
 
-    type_hints["return"] = _widget_return_type(info, unresolved_inputs)
+    type_hints["return"] = List[Layer]
```

Comment:
I'd rather not hardcode in the type, here - can we make use of the function signature?

Reply:
Cleaned up in 42e0463